packages/visualizations-react/stories/Poi/PoiMap.stories.tsx

```diff
@@ -170,3 +170,18 @@ const PoiMapLegendCenterArgs = {
     options: { ...options, legend: { ...legend, align: 'center' as const } },
 };
 PoiMapLegendCenter.args = PoiMapLegendCenterArgs;
+
+/**
+ * STORY: With legend on center align
+ */
+export const PoiMapMinMaxZooms: ComponentStory<typeof PoiMap> = Template.bind({});
+const PoiMapMinMaxZoomsArgs = {
+    data: { value: { layers: [{ ...layer1, colorMatch: citiesColorMatch }, layer2], sources } },
+    options: { 
+        ...options,
+        legend: { ...legend, align: 'center' as const },
+        minZoom: 3,
+        maxZoom: 5,
+    },
+};
+PoiMapMinMaxZooms.args = PoiMapMinMaxZoomsArgs;
```

Comment:
```suggestion
/**
 * STORY: With min and max zoom
 */
export const PoiMapMinMaxZooms: ComponentStory<typeof PoiMap> = Template.bind({});
const PoiMapMinMaxZoomsArgs = {
    data: { value: { layers: [{ ...layer1, colorMatch: citiesColorMatch }, layer2], sources } },
    options: { 
        ...options,
        legend,
        minZoom: 3,
        maxZoom: 5,
    },
};
PoiMapMinMaxZooms.args = PoiMapMinMaxZoomsArgs;
```

No need to override the legend style